<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatSettingsWidget</class>
 <widget class="QWidget" name="ChatSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0" columnstretch="0">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>406</width>
        <height>565</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="tabbedBox">
            <property name="title">
             <string>Tabbed mode</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QCheckBox" name="chatConfBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Chats and conferences in one window</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="closeButtonBox">
               <property name="text">
                <string>Close button on tabs</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="movableBox">
               <property name="text">
                <string>Tabs are movable</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="rememberBox">
               <property name="text">
                <string>Remember openned privates after closing chat window</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="closeOnSendBox">
            <property name="text">
             <string>Close tab/message window after sending a message</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="openOnNewBox">
            <property name="text">
             <string>Open tab/message window if received message</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="webkitBox">
            <property name="text">
             <string>Webkit mode</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="eventsBox">
            <property name="text">
             <string>Don't show events if message window is open</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="enterBox">
            <property name="text">
             <string>Send message on enter</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QCheckBox" name="doubleBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Send message on double enter</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="typingBox">
            <property name="text">
             <string>Send typing notifications</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="blinkBox">
            <property name="text">
             <string>Don't blink in task bar</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <widget class="QCheckBox" name="showAllBox">
            <property name="text">
             <string>After clicking on tray icon show all unreaded messages</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="removeBox">
              <property name="text">
               <string>Remove messages from chat view after:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="countSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>800</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="12" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="title">
             <string>Text browser mode</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0" colspan="2">
              <widget class="QCheckBox" name="namesBox">
               <property name="text">
                <string>Show names</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Timestamp:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="timeComboBox">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <item>
                <property name="text">
                 <string>( hour:min:sec day/month/year )</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>( hour:min:sec )</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>( hour:min, full date )</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="13" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>213</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="9" column="0">
           <widget class="QCheckBox" name="colorizeBox">
            <property name="text">
             <string>Colorize nicknames in conferences</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>removeBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>countSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabbedBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>chatConfBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enterBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>enterBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
